src/pathways/services/pathwayUtils.service.ts

```diff
@@ -0,0 +1,151 @@
+import * as _ from 'lodash';
+import * as fp from 'lodash/fp';
+import * as jsnx from 'jsnetworkx';
+
+export class PathwayUtilsService {
+    // Move to Escher once it's functional
+    DEFAULT_COFACTORS = new Set([
+        'MNXM3',  // 'ATP'
+        'MNXM7',  // 'ADP'
+        'MNXM8',  // 'NAD(+)'
+        'MNXM10',  // 'NADH(2-)'
+        'MNXM5',  // 'NADP(+)
+        'MNXM6',  // 'NADPH'
+        'MNXM51',  // 'GTP'
+        'MNXM30',  // 'GDP'
+        'MNXM12',  // 'CoA'
+        'MNXM80',  // 'UMP(2-)'
+        'MNXM1',  // 'H(+)'
+        'MNXM4',  // 'O2'
+        'MNXM13',  // 'CO(2)'
+        'MNXM2',  // 'H2O'
+    ]);
+
+    // Utility function since the Set interface is rather poor at the moment
+    public setUnion(...sets) {
```

Comment:
Unfortunately TS does not yet support the spread operator for `Set`. I could work it around by using 

    new Set([...Array.from(setA), ...Array.from(setB)])

which I find a bit too verbose. My reasoning was that implementing a union function for Sets with such a nice interface (arbitrary number of parameters) will be useful over the long run. But for now I inlined it with the Array conversion workaround.